Agrega pruebas cypress

YamiRampello opened the pull request:
Se agregan pruebas de integración en cypress

Commit:
Agrega pruebas cypress

Se agregan pruebas de integración en cypress

Commit:
Cambia la ruta de jquery

Al probar el deploy en github, tiraba un error porque buscaba el archivo de jquery de manera local

anarod13 commented on cypress/e2e/pruebas-interfaz.cy.js:
```diff
@@ -0,0 +1,44 @@
+
+    it('se asegura que los tipos de cambio no esten visibles', () => {
+      cy.visit(URL);
+      cy.get('#lista-datos').should('not.be.visible');
```
Recuerda evitar términos como info, datos o data en tus variables o clases ya que no brindan información específica sobre el elemento, podes optar por algo más descriptivo como `#lista-tipos-de-cambio`

anarod13 reviewed (approved):
Este test te quedó mucho más completo! Y está todo ok.

En cuanto al proyecto recuerda agregar validaciones a tu formulario antes de submitearlo, para evitar que se rompa, hacer requests innecesarios o renderear valores undefined

![imagen](https://github.com/YamiRampello/casa-de-cambio/assets/71040644/d91a2f6d-1133-409f-afc9-bed0182a141a)

YamiRampello committed:
Modifica nombres id

En función de las correcciones recibidas, se cambian los id para que sean más específicos

YamiRampello committed:
Modifica ruta para que no se vea la IP

En función de las correcciones recibidas, se establece una ruta base que se configura como una variable. Así, no queda escrita en la prueba pero cypress sabe donde encontrar la página
